deploy/cbdb_deploy.sh

```diff
@@ -71,7 +71,7 @@ function cbdb_build() {
 
     #download dbgen
     cd ~/workspace
-    curl https://artifactory.hashdata.xyz/artifactory/greenplum/cbdb/tool/dbgen/dbgen.tar.gz -o dbgen.tar.gz
+    curl https://cbdb-deps.s3.amazonaws.com/dbgen.tar.gz -o dbgen.tar.gz
```

Comment:
What's the use case of the tool `dbgen`? For TPC benchmark? If needed, we should download it from its official website.